addons/l10n_kz/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Kazakhstan - Accounting",
+    "version": "1.0.0",
+    "category": "Accounting/Localizations/Account Charts",
+    "license": "LGPL-3",
+    "depends": [
+        "account",
+    ],
+    "data": [
+        # Chart of Accounts
+        "data/account_chart_template_data.xml",
+        #"data/account_account_tag_data.xml",
+        "data/account_account_template_data.xml",
+        "data/account.group.template.csv",
+
+        # Taxes
+        #"data/account_tax_group_data.xml",
+        #"data/account_tax_report_data.xml",
+        #"data/account_tax_template_data.xml",
```

Comment:
The taxes should really be part of the templates before releasing the first version of the module